src/IW/App/Error.hs

```diff
@@ -118,7 +123,20 @@ data AppErrorType
     | UrlDownloadFailed Url
     deriving (Show, Eq)
 
--- | Map the @github@ library's @Error@ type into AppErrorType.
+type CabalErrorInfo = (Maybe Version, [CabalPError])
+
+{- | A wrapper around the 'PError' type from the @Cabal@ library.
+This is needed to implement an instance of 'Eq' for 'PError' without
+triggering the @orphan-instances@ warning. -}
+newtype CabalPError = CabalPError
+    { unCabalPError :: PError
+    } deriving newtype (Show)
+
+instance Eq CabalPError where
+    CabalPError (PError pos1 str1) == CabalPError (PError pos2 str2) =
+        pos1 == pos2 && str1 == str2
+
+-- | Map the @github@ library's 'Error' type into 'AppErrorType'.
```

Comment:
```suggestion
-- | Map the @github@ library's 'GitHub.Error' type into 'AppErrorType'.
```
(it would be displayed as just `Error`, otherwise, it is not in scope.